app/globals.css

```
@@ -43,7 +43,7 @@
         font-display: swap;
     }
 
-    *:not(.codicon) {
+    *:not(.codicon):not(code):not(pre):not(code *):not(pre *):not(.monospace):not(.monospace *) {
```

Stylelint (17.4.0)

[error] Expected complex :not() pseudo-class notation (selector-not-notation)

(selector-not-notation)